HISTORY.md

```diff
@@ -1,5 +1,15 @@
 # Release History - open AEA
 
+## 1.35.0 (2023-06-20)
+
+Plugins:
+- Adds support for multiple transaction builders on `flashbots` ledger plugin
+- Pins
+  - `eth-account`to `>=0.5.9,<0.6.0`
```

Comment:
The eth_account version with the bytes32 fix is 0.6.1, so this will be a problem.

Reply:
See [here](https://github.com/ethereum/eth-account/issues/90#issuecomment-1050317881)

Reply:
flashbots doesnt support that version

Reply:
Ok, I'll keep the custom version on governatooorr for now. Viraj, please dont close the open issue then.